content/en/docs/tasks/administer-cluster/securing-a-cluster.md

```diff
@@ -4,6 +4,7 @@ reviewers:
 - liggitt
 - ericchiang
 - destijl
```

Comment:
Remove destijl and ericchiang, they are no longer involved in upstream Kubernetes.